QUEST_JOBSTEP.tsv

```diff
@@ -61,36 +61,36 @@ QUEST_JOBSTEP_20150317_000060	Oh, this. I saw it somewhere...hmm...looks familia
 QUEST_JOBSTEP_20150317_000072	This is of good quality. The Hunter Master will be pleased. {nl}You can take the book. {nl}
-QUEST_JOBSTEP_20150317_000073	Merogs are in the Fallen Worry Forest, and the Quarrel Shooter Master is in Klaipeda. {nl}Someone might get ahead of you so better hurry.
-QUEST_JOBSTEP_20150317_000074	The Archer Master in Klaipeda has the next book. {nl}He's not a mean person so he will give you the book if you ask nicely. {nl}
-QUEST_JOBSTEP_20150317_000075	He is helping with the repair of the damaged city wall. {nl}I heard there are rocks delivered to the Miner's Village, so bringing him that rock can be a way to ask. 
+QUEST_JOBSTEP_20150317_000073	$Merogs are in the Fallen Worry Forest, and the Quarrel Shooter Master is in Klaipeda. {nl}Someone might get ahead of you, so better hurry.
+QUEST_JOBSTEP_20150317_000074	$The Archer Master in Klaipeda has the next book. {nl}He's not a mean person, so he will give you the book if you ask nicely. {nl}
+QUEST_JOBSTEP_20150317_000075	$He is helping with the repair of the damaged city wall. {nl}I heard rocks were delivered to the Miners' Village, and bringing him a rock can be a way to ask. 
 QUEST_JOBSTEP_20150317_000076	Did the Hunter Master send this rock? {nl}If it's Lydia Schaffen's book, you could have just asked. But thank you anyway. 
-QUEST_JOBSTEP_20150317_000077	The last book is with the Ranger Master in Klaipeda. {nl}Give her this pouch. {nl}I could go myself but we're not really comfortable seeing each other. 
+QUEST_JOBSTEP_20150317_000077	$The last book is with the Ranger Master in Klaipeda. {nl}Give her this pouch. {nl}I could go myself, but we're not really comfortable seeing each other. 
 QUEST_JOBSTEP_20150317_000078	Don't ask why. Work is work, and it's a fact that we don't want to see each other. 
 QUEST_JOBSTEP_20150317_000079	This pouch...Did you come from the Hunter Master? {nl}Here. Give her this book. 
 QUEST_JOBSTEP_20150317_000080	I don't want to act mean when she's acting humble. 
 QUEST_JOBSTEP_20150317_000081	This leather will be enough. {nl}Go to the Quarrel Shooter Master before someone else takes it. 
 QUEST_JOBSTEP_20150317_000082	This rock will be useful in repairing the wall. {nl}Bring it to the Archer Master quickly. 
-QUEST_JOBSTEP_20150317_000083	Are you the one who wants to learn my skills? {nl}Great, I was just looking for someone to help me find Lydia Schaffen's book. {nl}You're going to help right? 
-QUEST_JOBSTEP_20150317_000084	There is one in the Miner's Village but it was lost when the Vubbes raided. {nl} I'm sure it's in the Vubbe Outpost.
+QUEST_JOBSTEP_20150317_000083	$Are you the one who wants to learn my skills? {nl}Great, I was just looking for someone to help me find Lydia Schaffen's book. {nl}You're going to help, right? 
+QUEST_JOBSTEP_20150317_000084	$There is one in the Miners' Village, but it was lost when the Vubbes raided. {nl} I'm sure it's in the Vubbe Outpost.
```

Comment:
Tense? If it's not there anymore, then it *was* there, not *is* there.
"There was one in the Miners' Village, but it was lost when the Vubbes raided."